### ISSUE TITLE
Issue with boolean parse.

### ISSUE THREAD

**Issue by dariobrux** [flipkart-incubator/proteus#145]
I'm using the plugin with a ViewGroup setting the ClipToPadding property to "false" (string) value.
For this property, the parser returns a true value (Boolean). 
Inspecting the code, I found that the problem seems to be in this function:

```
    public static boolean parseBoolean(@Nullable Value value)
    {
        // TODO: we should consider 0 as false too.
        return null != value && value.isPrimitive() && value.getAsPrimitive().isBoolean() ? value.getAsBoolean() : null != value && !value.isNull();
    }
```

Since the value is passed as String, the function goes in the second part of the return statement:
`null != value && !value.isNull()`

Here, since the value is not null, the function returns the true Boolean value even if the string was "false".

I changed the function to the following:
```
    public static boolean parseBoolean(@Nullable Value value)
    {
        // TODO: we should consider 0 as false too.
        return null != value && value.isPrimitive() && value.getAsPrimitive().isBoolean() ? value.getAsBoolean() : null != value && !value.isNull() && Boolean.parseBoolean(value.getAsString());
    }
```

Now everything works fine.

I was wondering why nobody had this problem.

Thank you,

Dario Brux

**Comment by adityasharat** [comment]
Well I would guess that in most cases people would pass it as a boolean instead of a string.

```
{
  "property": false
}
```

That being said this is a welcome change, please consider raising a PR for this.

**Comment by adityasharat** [comment]
Closed by #146

**Comment by adityasharat** [comment]
Available since [v5.0.1](https://github.com/flipkart-incubator/proteus/releases/tag/5.0.1)
